docs/custom_service.md

```diff
@@ -1,128 +1,133 @@
-               response[output[0][i]] = self.labels[i]
-           return response
-```
+MMS, identifies the entry point to the custom service, from the manifest file. Thus file exporting the model archive, one needs to mention the entry point using the ```--handler``` option. 
 
-You might be thinking, wait a minute, I thought we used the vision service for the resnet-18 example. Yes, that's right, but what we're showing here is how you might extend the base service and manually define the image size and the number of outputs rather than rely on the vision service.
-
-### An Object Detection Example
-
-Another more complex example using object detection (SSD) is provided in the [examples/ssd](../examples/ssd/README.md) folder. The goal of this example is to use an object detection model to detect several objects in the image, classify them, and also return bounding boxes to show where in the image the objects were detected. As opposed to the resnet-18 example, we take advantage of the vision service and extend it slightly to record a copy of the original image's shape during pre-processing.
+The [model-archiver](https://github.com/awslabs/mxnet-model-server/blob/master/model-archiver/README.md) tool enables the export to a format understood by MMS.
 
 ```python
-def _preprocess(self, data):
-    input_image = image.read(data[0])
-    self.input_height = input_image.shape[0]
-    self.input_width = input_image.shape[1]
-    return super(SSDService, self)._preprocess(data)
+model-archiver --model-name <model-name> --handler custom_service:execute --export-path <export-dir> --model-path <model_dir>
 ```
 
-We then use this info in post-processing to provide accurate bounding boxes.
-
-```python
-def _postprocess(self, data):
-    detections = data[0].asnumpy()
-    result = []
-    for i in range(detections.shape[0]):
-        det = detections[i, :, :]
-        res = det[np.where(det[:, 0] >= 0)[0]]
-        result.append(res)
-
-    dets = result[0]
-    classes = self.labels
-    width = self.input_width    # original input image width
-    height = self.input_height  # original input image height
-    response = []
-    for i in range(dets.shape[0]):
-        cls_id = int(dets[i, 0])
-        if cls_id >= 0:
-            score = dets[i, 1]
-            if score > self.threshold:
-                xmin = int(dets[i, 2] * width)
-                ymin = int(dets[i, 3] * height)
-                xmax = int(dets[i, 4] * width)
-                ymax = int(dets[i, 5] * height)
-                class_name = str(cls_id)
-                if classes and len(classes) > cls_id:
-                    class_name = classes[cls_id]
-                response.append((class_name, xmin, ymin, xmax, ymax))
-    return response
+This will create file ```<model-name>.mar``` in the directory ```<export-dir>```
+
+This will create a model archive with the custom handler.
+
+The manifest file of the generated model archive will have the following content.
```

Comment:
The created MANIFEST.json doesn't have all the fields. We don't need to expose MANIFEST.json format for now.